### PR TITLE
content: add open source communities blog

### PR DESCRIPTION
**Pull request by lukegalbraithrussell**
This pull request creates a new blog "Where Should Open Source Communities Chat?"

### CODE REVIEW

File: src/content/learn/blog/where_should_open_source_communities_chat.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+title: 'Where Should Open Source Communities Chat?'
+date: 2020-08-04
+description: 'Open source communities can chat with GitHub Discussions, but there are other options as well.'
+metaTitle: 'Chatting with Open Source Communities.'
+author: 'Matthew Gregory'
```
Comment (mattgreg) [inline src/content/learn/blog/where_should_open_source_communities_chat.md:6]: update author to your name

File: src/content/learn/blog/where_should_open_source_communities_chat.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+authorAvatar: ./assets/matthew_gregory-1.png
```
Comment (mattgreg) [inline src/content/learn/blog/where_should_open_source_communities_chat.md:7]: create a headshot avitar. The style is white background and a B&W photo. put it here: https://github.com/ockam-network/website/tree/production/src/content/learn/blog/assets

File: src/content/learn/blog/where_should_open_source_communities_chat.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+isFeature: true
```
Comment (mattgreg) [inline src/content/learn/blog/where_should_open_source_communities_chat.md:8]: remove line 8

**Comment by mattgreg** [comment]
recheck cla

**Comment by lukegalbraithrussell** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by mattgreg** [comment]
@lukegalbraithrussell can you squash this commit history to 1 commit that is only the blog addition? (remove chore: bump lodash)

**Comment by mattgreg** [comment]
@lukegalbraithrussell May I suggest that you close this PR, reclone the develop branch, commit this blog and the /assets files as a new branch and then submit a new PR? That's probably the easiest way to solve the merge here.

**Pull request closed without merging**